doc/architecture/sensor_view.adoc

```diff
@@ -1,9 +1,8 @@
 = Sensor view
 
-Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
-Et malesuada fames ac turpis egestas.
-Mauris pharetra et ultrices neque ornare aenean euismod elementum nisi.
-Nulla facilisi morbi tempus iaculis urna id. Justo eget magna fermentum iaculis.
-Sed augue lacus viverra vitae congue eu. Velit ut tortor pretium viverra suspendisse.
-Amet commodo nulla facilisi nullam vehicula ipsum a. Nibh nisl condimentum id venenatis a.
-Diam vel quam elementum pulvinar etiam non quam lacus suspendisse.
\ No newline at end of file
+Sensor view imitates the input of sensors.
+Sensor-view data is derived from ground-truth data.
+All information regarding the environment is given with respect to the virtual sensor coordinate system, with two exceptions:
+
+* Individual physical technology-specific data, given with respect to the physical sensor coordinate system specified in the corresponding physical sensor’s mounting position
+* Ground truth data, given in global coordinates.
```

Comment:
Any specific reason why in some places it is mentioned as "ground truth" and in others "ground**-**truth"